Front/details

andres-rodriguez16 opened the pull request:
updating styles of the detail of each video game

andres-rodriguez16 committed:
refactor(front/navBar/styles)

andres-rodriguez16 committed:
update(search/styles)

andres-rodriguez16 committed:
refactor(home/filters)

andres-rodriguez16 committed:
update(filters styles)

andres-rodriguez16 committed:
Merge branch 'front/navBar' of https://github.com/andresyrg16/videogames into front/details

andres-rodriguez16 committed:
cambios

andres-rodriguez16 committed:
home

andres-rodriguez16 committed:
refactor/details

andres-rodriguez16 committed:
refactor details

andres-rodriguez16 committed:
refactor details

nelsoncardenas commented on api/src/routes/rutaPorId.js:
```diff
@@ -11,13 +11,42 @@ router.get("/:idVi", async (req, res) => {
         include : Genero
       })
-      res.send(busquefaDB)
+      // console.log(busquefaDB.dataValues.Generos.map(el => el.name));
```
Remove comment
```suggestion
```

nelsoncardenas commented on api/src/utils/busquedaPorID.js:
```diff
@@ -5,20 +5,41 @@ async function getGameId(idVi) {
+      platforms: searchPorId.data.platforms?.map(p => p.platform.name)
+    };
+    let genres = ""
+    game.Generos.forEach((el, i) => {
```
you should avoid using names like `el`. Try to use meaningful names.

nelsoncardenas commented on api/src/utils/busquedaPorID.js:
```diff
@@ -5,20 +5,41 @@ async function getGameId(idVi) {
+    game.Generos = genres
+
+    let platform = ""
+    game.platforms.forEach((el, i) => {
```
change `el` name

nelsoncardenas commented on api/src/utils/fecthAPIGamesget.js:
```diff
@@ -14,6 +14,8 @@ async function fecthAPIGames(url, count) {
 
 function datosTraidosDeLaApi(datos) {
   try {
+    // datos.genres.replace("Action", "hola")
```
Remove unnecesary comment

yulianacastrodiaz replied:
remove comment, this conversation not resolved

nelsoncardenas commented on client/src/App.module.css:
```diff
@@ -15,4 +15,25 @@
+    height: auto;
+    display: flex;
+    justify-content: center;
+    /* align-items: center; */
```
another comment

yulianacastrodiaz replied:
idem

nelsoncardenas commented on client/src/componets/Details/Details.module.css:
```diff
@@ -1,66 +1,141 @@
+  color: #FE7F3C;
 }
+
+.lanzamiento {
```
You are not using english. Maybe you can create an issue for this one instead of changing it for this PR.

nelsoncardenas replied:
@andresyrg16  I'm still waiting for the issue creation

nelsoncardenas commented on client/src/componets/Details/Details.module.css:
```diff
@@ -1,66 +1,141 @@
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
```
Unnecessary space.

nelsoncardenas commented on client/src/componets/Formulario/Plataformas.js:
```diff
@@ -18,4 +18,10 @@ const platforms = [
   'PlayStation 2',
   "Wii U"
 ] 
+// {detailsVideoGames.platforms.splice(detailsVideoGames.platforms.indexOf('Nintendo Switch'),detailsVideoGames.platforms.indexOf('Nintendo Switch'), "https://cdn-icons-png.flaticon.com/512/871/871377.png" )}
```
Why did you add those comments?

nelsoncardenas commented on client/src/componets/Navbar/Navbar.jsx:
```diff
@@ -20,7 +20,7 @@ const NavBar = () => {
         <section className={s['navbar--section']}>
           <ul className={s['navbar--section__container']}>
-            <li>
+            <li >
```
Unnecessary extra space

nelsoncardenas commented on client/src/componets/filters/Filters.jsx:
```diff
@@ -0,0 +1,80 @@
+    dispatch(ratingSort(e.target.value));
+    setActualPage(1)
+  }
+  function handleFiltroPorAlpha(e) {
```
add break line

nelsoncardenas replied:
You didn't solve the problem. Evidence:
![image](https://user-images.githubusercontent.com/18086414/191272700-4e7a4c72-d2c1-4f55-afed-aee7048d2b47.png)

nelsoncardenas commented on client/src/componets/filters/Filters.jsx:
```diff
@@ -0,0 +1,80 @@
+  function handleFiltroPorAlpha(e) {
+    dispatch(filtroPorAlpha(e.target.value));
+    setActualPage(1)
+  }
```
add break line

nelsoncardenas replied:
You didn't solve the problem. Evidence:
![image](https://user-images.githubusercontent.com/18086414/191272700-4e7a4c72-d2c1-4f55-afed-aee7048d2b47.png)

andres-rodriguez16 committed:
Update rutaPorId.js

  resolved  change el name, you should avoid using names like el. Try to use meaningful names, Remove comment

andres-rodriguez16 committed:
update style form

andres-rodriguez16 committed:
deploy back

andres-rodriguez16 committed:
refactoring error code

andresyrg16 commented:
recommendations corrected, ready to revisit

andres-rodriguez16 committed:
deploy v1

andres-rodriguez16 committed:
deploy v2

yulianacastrodiaz commented on api/src/app.js:
```diff
@@ -9,12 +9,13 @@ require('./db.js');
 const server = express();
 
 server.name = 'API';
+// 'http://localhost:3000'
```
remove comment

yulianacastrodiaz commented on api/src/db.js:
```diff
@@ -5,13 +5,38 @@ const path = require('path');
+    dialectOptions: {
+      ssl: {
+        require: true,
+        // Ref.: https://github.com/brianc/node-postgres/issues/2009
```
remove comment

yulianacastrodiaz commented on api/src/db.js:
```diff
@@ -5,13 +5,38 @@ const path = require('path');
+  })
+    :
+    new Sequelize(`postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}/videogames`, {
+      logging: false, // set to console.log to see the raw SQL queries
```
idem

yulianacastrodiaz commented on api/src/db.js:
```diff
@@ -5,13 +5,38 @@ const path = require('path');
+    :
+    new Sequelize(`postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}/videogames`, {
+      logging: false, // set to console.log to see the raw SQL queries
+      native: false, // lets Sequelize know we can use pg-native for ~30% more speed
```
idem

yulianacastrodiaz commented on api/src/utils/busquedaPorID.js:
```diff
@@ -5,20 +5,41 @@ async function getGameId(idVi) {
-  
+
 }
 module.exports = { getGameId };
```
missing break line, please review code quality

yulianacastrodiaz commented on client/src/App.module.css:
```diff
@@ -15,4 +15,25 @@
+    width: 80%;
+    margin: 50px 50px;
+  }
+}
```
missing break line, please review the code quality

yulianacastrodiaz commented on client/src/componets/Details/Details.module.css:
```diff
@@ -1,66 +1,132 @@
+  .platforms {
+    font-family: 'Montserrat', sans-serif;
+  }
 }
```
missing break line, please review the code quality

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
-  color:  #FE7F3C; /* color de fuente */
-  border: 2px solid  #FE7F3C; /* tamaño y color de borde */
+  background: #fff;
+  /* color de fondo */
```
remove comment

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
+  background: #fff;
+  /* color de fondo */
+  color: #FE7F3C;
+  /* color de fuente */
```
idem

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
+  color: #FE7F3C;
+  /* color de fuente */
+  border: 2px solid #FE7F3C;
+  /* tamaño y color de borde */
```
idem

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
   padding: 16px 40px;
-  border-radius: 3px; /* redondear bordes */
+  border-radius: 3px;
+  /* redondear bordes */
```
idem

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
   z-index: 1;
   overflow: hidden;
-  margin-left: 40%;
+  /* margin-left: 40%; */
```
idem

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
-  color: #fff; /* color de fuente hover */
+.button__submit:hover {
+  color: #fff;
+  /* color de fuente hover */
```
idem

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -25,29 +15,40 @@
   content: "";
-  background:  #FE7F3C; /* color de fondo hover */
+  background: #FE7F3C;
+  /* color de fondo hover */
```
idem

yulianacastrodiaz commented on client/src/componets/Navbar/Navbar.module.css:
```diff
@@ -52,35 +52,58 @@ a:focus {
+   margin-right: 10px;
+   margin-left: 10px;
+  }
+}
```
missing break line, please review the code quality

yulianacastrodiaz commented on client/src/redux/actions/action.js:
```diff
@@ -14,115 +14,117 @@ export const ORDEN_FECHA = "ORDEN_FECHA"
+    type: ORDEN_FECHA,
+    payload: orden
   }
 }
```
missing break line, please review the code quality

andres-rodriguez16 committed:
solving recommendations

andresyrg16 commented:
recommendations checklist

andres-rodriguez16 committed:
solving recommendations

yulianacastrodiaz commented on api/src/routes/rutaPorGames.js:
```diff
@@ -80,37 +78,35 @@ router.post("/", async (req, res) => {
+  } catch (error) {
+  }
 })
 module.exports = router;
```
missing break line

yulianacastrodiaz commented on client/src/componets/Formulario/Formulario.module.css:
```diff
@@ -109,12 +116,62 @@
+    margin-top: 20px;
+    margin-bottom: -20px;
+  }
 }
```
missing break line

yulianacastrodiaz commented on client/src/componets/Navbar/Navbar.module.css:
```diff
@@ -52,35 +52,56 @@ a:focus {
+    margin-right: 10px;
+    margin-left: 10px;
+  }
+};
```
missing break line

yulianacastrodiaz commented on client/src/redux/actions/action.js:
```diff
@@ -14,115 +14,117 @@ export const ORDEN_FECHA = "ORDEN_FECHA"
-      })
-      .catch(e => console.log(e))
+      } catch (error) {
+        console.log(error); 
```
esto no debería tener un console.log, debe lanzar el error.

yulianacastrodiaz commented on client/src/redux/actions/action.js:
```diff
@@ -14,115 +14,117 @@ export const ORDEN_FECHA = "ORDEN_FECHA"
-      })
-      .catch(e => console.log(e))
+    } catch (error) {
+      console.log(error)
```
idem

yulianacastrodiaz commented on client/src/redux/actions/action.js:
```diff
@@ -14,115 +14,117 @@ export const ORDEN_FECHA = "ORDEN_FECHA"
   }
-}
\ No newline at end of file
+};
```
missing break line

andres-rodriguez16 committed:
resolviendo recomendaciones

yulianacastrodiaz commented on api/src/routes/rutaPorId.js:
```diff
@@ -11,13 +11,41 @@ router.get("/:idVi", async (req, res) => {
+      res.send(videoGame)
     }
   } catch (error) {
     console.log(error)
```
el catch, debe lanzar un error, no hacerle console.log

yulianacastrodiaz commented on api/src/utils/busquedaPorID.js:
```diff
@@ -5,20 +5,39 @@ async function getGameId(idVi) {
+    game.platforms = platforms
+    return game;
   } catch (error) {
     return error;
```
el catch lanza un error, no lo retorna, lo lanza

yulianacastrodiaz commented on client/src/redux/actions/action.js:
```diff
@@ -14,115 +14,117 @@ export const ORDEN_FECHA = "ORDEN_FECHA"
+        payload: response.data
+      });
+    } catch (error) {
+      console.log(error);
```
controlar el error

yulianacastrodiaz commented on client/src/redux/actions/action.js:
```diff
@@ -14,115 +14,117 @@ export const ORDEN_FECHA = "ORDEN_FECHA"
-      })
-      .catch(e => console.log(e))
+      } catch (error) {
+        return error; 
```
controlar el error, no retornarlo

andres-rodriguez16 committed:
quitando console.log

yulianacastrodiaz commented on api/src/routes/rutaPorGames.js:
```diff
@@ -80,37 +78,35 @@ router.post("/", async (req, res) => {
+        }
+      })
+    res.send("videogames actualizado")
+  } catch (error) {
```
controlar el error

andres-rodriguez16 committed:
add breaÃk line

andres-rodriguez16 committed:
eliminado console.log, colocando throw

andres-rodriguez16 committed:
resolviendo recomendaciones

nelsoncardenas commented:
I noticed you have the path `client/src/componets/filters/Filters.jsx`, but looking at the components folder, I can see you use Capitalized folders:

![image](https://user-images.githubusercontent.com/18086414/191271962-69d1d1ce-c6c4-4060-bd12-0d946e42260c.png)

So I think maybe you have two errors in two folders. `landingPage` and `filters`. What do you think?

nelsoncardenas commented:
Are filters a component or a helper?

andres-rodriguez16 committed:
modificando el paginado

andres-rodriguez16 committed:
volviendo al paginado anterior

andres-rodriguez16 committed:
acomodando los filtros

andres-rodriguez16 committed:
finishing corrections

nelsoncardenas commented on client/src/componets/Paginado/Paginado.module.css:
```diff
@@ -38,8 +38,3 @@
-/* .paginado button:hover {
-  background-color: rgb(255, 255, 255);
-  color: #FE7F3C
-} */
```
Missing break line

andres-rodriguez16 committed:
update(cliete/Paginado.module.css)

andres-rodriguez16 committed:
deploy rute
